app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -135,6 +151,10 @@
 }
 
 .gem-c-image-card--large.gem-c-image-card {
+  display: flex;
+  @include govuk-media-query($from: tablet) {
```

Comment:
Can we keep the `display` and `margin` selectors together above the `@include` bit?

Reply:
Fixed